src/spatial/collision.js

```diff
@@ -78,17 +75,15 @@ Crafty.c("Collision", {
      *
      * @see Crafty.polygon
      */
-    collision: function (poly) {
+    collision: function (polygon) {
```

Comment:
I renamed this argument like if described [here](https://github.com/craftyjs/Crafty/blob/develop/src/spatial/collision.js#L48). You don't mind?